lua/ttt2/libraries/modelbuilder.lua

```diff
@@ -0,0 +1,316 @@
+---
+-- Used to build custom world and view models. 
```

Comment:
Also rendering.

But please also describe that this needs the weapon base to contain specific entries for this to work.

Maybe we need to also add a comment to all `wep` params that this only works with tttbase weapons.

Reply:
Agreed. Regarding the last point: Don't we already assume that all weapons in TTT are based on the ttt weapon base?

Reply:
Mhh fair, but still mention it, because it helps with understanding that this can only be used on weapon entities and is tightly coupled to some fields and functions therein

Reply:
I think you missed this

Reply:
<img width="644" alt="image" src="https://github.com/TTT-2/TTT2/assets/13639408/3a1b489f-b13a-4e62-bb32-b386fd1a07bb">

not, it is updated